script.js

```diff
@@ -0,0 +1,236 @@
+// api url
+const API_URL =
+	'https://raw.githubusercontent.com/alexsimkovich/patronage/main/api/data.json';
+
+// declaration of items creating product list
+let $ulList;
```

Comment:
w vanilla JS nie używamy `$` w nazwach zmiennych (tak jak w jQuery).

Reply:
Usunięto.